doc/build.md

```diff
@@ -110,6 +110,8 @@ BUILD_PXR_PLUGIN            | builds the Pixar USD plugin and libraries.
 BUILD_AL_PLUGIN             | builds the Animal Logic USD plugin and libraries.          | ON
 BUILD_HDMAYA                | builds the Maya-To-Hydra plugin and scene delegate.        | ON
 BUILD_TESTS                 | builds all unit tests.                                     | ON
+BUILD_STRICT_MODE           | enforces all warnings as errors.                           | ON
+BUILD_SHARED_LIBS			| build libraries as shared or static.						 | ON
```

Comment:
Funky whitespacing. Tab characters, maybe?

Reply:
Hey @mattyjams , indeed funky whitespacing but one thing I have noticed is that is how github shows it when it wraps charcaters.  In my text editor, I usually have word wrap off when I work with *.md file.

I have also noticed that sometimes github doesn't properly visualize spaces vs tabs.